orttraining/orttraining/eager/opgen/opgen/generator.py

```diff
@@ -239,12 +239,14 @@ def _write_cpu_fall_back(self, writer: opgenwriter.SourceWriter, mapped_func: Ma
         writer.writeline(");")
         writer.pop_indent()
 
-    def _write_function_body_debug_header(self, writer, func_parameters):
-        # Debug Logging
+    # Generates a log line for method entry, with the function parameters.
+    def _write_function_body_entry_logging(self, writer, func_parameters):
```

Comment:
Prefer using docstrings. See https://google.github.io/styleguide/pyguide.html#383-functions-and-methods